cfpb/cfgov-refresh
# Remove Info Unit Group from Blog pages, restrict to 25/75 layout on Learn pages

When Info Unit Groups were added recently, I overlooked the fact that we needed the ability to disallow particular layouts on certain page types. This PR removes them entirely from Blog pages, and restricts the layout options to 25/75 only on a Learn page.

## Additions

- `InfoUnitGroup2575Only`, a subclass of `InfoUnitGroup` that offers only 25/75 as a choice for the `format` field
- CSS to hide the automatically generated select menu for `format` when rendering the Wagtail interface for `InfoUnitGroup2575Only`

## Removals

- `InfoUnitGroup` removed as an available module on Blog pages

## Changes

- `InfoUnitGroup2575Only` traded for the standard `InfoUnitGroup` as an available module on Learn pages
- Some minor cleanup of the existing Heading Block CSS

## Testing

1. Pull branch
2. Migrate
3. Run server
4. Create/edit a Learn page
5. Add an Info Unit Group to the content area
6. Observe that you can not select a format other than 25/75
7. Create/edit a Blog page
8. Observe that you cannot add an Info Unit Group.

## Screenshots

This is what you see when adding an Info Unit Group to a Learn page:

![screen shot 2017-08-07 at 12 46 00](https://user-images.githubusercontent.com/1044670/29941901-06f284da-8e62-11e7-8f56-5d5b1b029ac1.png)


## Checklist
